src/loaders/common/loader_helper.py

```diff
@@ -626,3 +626,14 @@ def create_meta_file(
 
     return metafile
 
+
+def obj_info_to_upa(obj_info: list[Any], underscore_sep=False) -> str:
```

Comment:
Actually can we put this in src/common vs src/loaders? I suspect we use it in the service as well

Reply:
👍

Reply:
:+1: